src/v2i-hub/CARMAStreetsPlugin/src/CARMAStreetsPlugin.h

```diff
@@ -48,6 +49,7 @@ class CARMAStreetsPlugin: public PluginClientClockAware {
 	void HandleMobilityOperationMessage(tsm3Message &msg, routeable_message &routeableMsg);
 	void HandleMobilityPathMessage(tsm2Message &msg, routeable_message &routeableMsg);
 	void HandleBasicSafetyMessage(BsmMessage &msg, routeable_message &routeableMsg);
+	void HandleSimulatedSensorDetectedMessage(simulation::SensorDetectedObject &msg, routeable_message &routeableMsg);
```

Comment:
Lets add a status field for sensor detected objects or at least add it to the manifest.json message types so it is generated automatically.

Reply:
Add the new SensorDetectedObject to the MessageTypes fields of the plugin manifest.json

Reply:
addressed